--- /usr/bin/linkfinder	2022-06-16 20:12:57.000000000 +0300
+++ linkfinder	2022-07-02 22:36:02.092861464 +0300
@@ -256,10 +256,10 @@ def html_save(html):
 
         print("URL to access output: file://%s" % os.path.abspath(args.output))
         file = "file:///%s" % os.path.abspath(args.output)
-        if sys.platform == 'linux' or sys.platform == 'linux2':
-            subprocess.call(["xdg-open", file])
-        else:
-            webbrowser.open(file)
+        #if sys.platform == 'linux' or sys.platform == 'linux2':
+        #    subprocess.call(["xdg-open", file])
+        #else:
+        #    webbrowser.open(file)
     except Exception as e:
         print("Output can't be saved in %s \
             due to exception: %s" % (args.output, e))
