From dfaac38e1d555c392e0e0cdff4ce92f9c4a47efe Mon Sep 17 00:00:00 2001
From: Zero2 0day <0x020d@gmail.com>
Date: Tue, 4 Jul 2017 22:12:46 -0400
Subject: [PATCH] Fixed timeout parameter

---
 smtp-user-enum.pl | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/smtp-user-enum.pl b/smtp-user-enum.pl
index 5613985..52ca21f 100755
--- a/smtp-user-enum.pl
+++ b/smtp-user-enum.pl
@@ -69,7 +69,7 @@
 	-T file  File of hostnames running the smtp service
 	-p port  TCP port on which smtp service runs (default: $smtp_port)
 	-d       Debugging output
-	-t n     Wait a maximum of n seconds for reply (default: $query_timeout)
+	-w n     Wait a maximum of n seconds for reply (default: $query_timeout)
 	-v       Verbose
 	-h       This help message
 
@@ -84,7 +84,7 @@
 
 USAGE
 
-getopts('m:u:U:s:S:r:dt:vhM:f:D:p:', \%opts);
+getopts('m:u:U:s:S:r:dt:vhM:f:D:p:w:', \%opts);
 
 # Print help message if required
 if ($opts{'h'}) {
@@ -105,6 +105,7 @@
 $smtp_port      = $opts{'p'} if $opts{'p'};
 $mode           = $opts{'M'} if $opts{'M'};
 $from_address   = $opts{'f'} if $opts{'f'};
+$query_timeout  = $opts{'w'} if $opts{'w'};
 
 # Check for illegal option combinations
 unless ((defined($username) or defined($username_file)) and (defined($host) or defined($host_file))) {
