From 724a1566709a3af15bf879348f085154ff458054 Mon Sep 17 00:00:00 2001
From: Alexandr Savca <alexandr.savca89@gmail.com>
Date: Sun, 30 Jul 2023 14:54:29 +0300
Subject: [PATCH 4/5] README: adjust to 80x24, fix libnet project address

---
 README | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/README b/README
index d8233bb..59d53dc 100644
--- a/README
+++ b/README
@@ -6,12 +6,12 @@
 Description
 -----------
 
-   This program sends out a custom ARP REPLY packet with the hardware and 
-protocol address information of your choosing. Since ARP is a stateless protocol, 
-most operating systems will gladly update their ARP cache with whatever
-information you send them in your hand-crafted packet.
+   This program sends out a custom ARP REPLY packet with the hardware and
+   protocol address information of your choosing.  Since ARP is a stateless
+   protocol, most operating systems will gladly update their ARP cache with
+   whatever information you send them in your hand-crafted packet.
  
 Notes
 -----
 
-   Requires Libnet -- http://www.packetfactory.net/Projects/Libnet/
+   Requires Libnet -- https://github.com/libnet/libnet
-- 
2.41.0

